components/TopBarProfileMenu.js

```diff
@@ -43,13 +46,89 @@ const HideGlobalScroll = createGlobalStyle`
 `;
 
 const UserMenuLinkEntry = props => {
+  const { isMobileMenuLink } = props;
   return (
-    <ListItem mb="6px">
+    <ListItem mb="6px" py={isMobileMenuLink ? '2' : '0'}>
       <StyledLink as={Link} fontWeight="500" fontSize="14px" lineHeight="20px" color="black.800" {...props} />
+      {isMobileMenuLink && <StyledHr mt={2} borderStyle="solid" borderColor="rgba(49, 50, 51, 0.1)" />}
     </ListItem>
   );
 };
 
+UserMenuLinkEntry.propTypes = {
+  isMobileMenuLink: PropTypes.bool,
+};
+
+const UserAccountLinks = props => {
+  const { setShowNewsAndUpdates, LoggedInUser, isMobileView, logOutHandler, height } = props;
+
+  return (
+    <Box height={height}>
+      {CHANGE_LOG_UPDATES_ENABLED && (
+        <UserMenuLinkEntry as={Span} isMobileMenuLink={isMobileView} onClick={() => setShowNewsAndUpdates(true)}>
+          <FormattedMessage id="menu.newsAndUpdates" defaultMessage="News and Updates" />
+        </UserMenuLinkEntry>
+      )}
+      <Query
+        query={memberInvitationsCountQuery}
+        variables={{ memberCollectiveId: LoggedInUser.CollectiveId }}
+        fetchPolicy="network-only"
```

Comment:
Since the other one already triggers the query, I think it's safe to rely on the cache here

```suggestion
```